src/data/data-gig-detail.html

```diff
@@ -1,12 +1,18 @@
 <link rel="import" href="../../bower_components/polymer/polymer.html">
 
 <link rel="import" href="../../bower_components/polymerfire/firebase-document.html">
+<link rel="import" href="data-gigs-list.html">
 
 
 <dom-module id="data-gig-detail">
 <template>
   <firebase-document id="document" app-name="polygigs"
     path="/gigs/[[gigId]]" data="{{raw}}"></firebase-document>
+
+  <!--<template is="dom-if" if="{{!document.online}}">-->
```

Comment:
May want to clarify what is intended here. I assume you just removed the `if` while testing and you do intend for `<data-gigs-list>` to exist. It kind of looks like you just forgot to comment it out.